content/python/concepts/built-in-functions/terms/memoryview/memoryview.md

```diff
@@ -0,0 +1,59 @@
+---
+Title: 'memoryview()'
+Description: 'Creates a 'memory view' object that allows Python code to access the internal data of an object's buffer without making a copy of it'
+Subjects:
+  - 'Data Types'
+  - 'Computer Science'
+  - 'Built-in Functions'
+Tags:
```

Comment:
You are missing tags in the metadata. Please choose two or three from [this list](https://github.com/Codecademy/docs/blob/main/documentation/tags.md).